### DISCUSSION TITLE
Ability to use something like BlissOS instead of Lineage for Waydroid

### DISCUSSION THREAD

**Discussion by VarLad** [waydroid/waydroid#597] [General]
I wonder if its possible to use BlissOS for android, instead of LineageOS
@aleasto Any ideas there?

**Comment by aleasto** [comment]
We are an open source project. You can compile whatever android fork you want with our patches applied.

The services we use for RPC between the host and the container are implemented on top of some LineageOS private apis, but it should be trivial to to port those to a generic AOSP source.
Specifically i believe BlissOS already implements the same apis.

**Comment by Schweeeeeeeeeeeeeeee** [comment]
Why isn't is a second option or a main option on desktop Linux? If it's not hard wouldn't that be a good feature?
